apps/src/sites/studio/pages/layouts/_thank_donors_interstitial.js

```diff
@@ -0,0 +1,24 @@
+import $ from 'jquery';
+
+$(document).ready(function() {
+  $('#dismiss-thank-donors').click(function() {
+    $('#thank-donors-modal').modal('hide');
+  });
+});
+
+function setCookie(key, value) {
+  document.cookie = key + '=' + value;
+}
+
+function getCookie(key) {
+  var keyValue = document.cookie.match('(^|;) ?' + key + '=([^;]*)(;|$)');
+  return keyValue ? keyValue[2] : null;
+}
+
+$(document).ready(function() {
+  var already_shown = !!getCookie('hide_thank_donors');
+  if (!already_shown && window.innerWidth > 720) {
+    $('#thank-donors-modal').modal('show');
+    setCookie('hide_thank_donors', '1');
```

Comment:
Let's name this cookie something like `has_seen_thank_donors`.  I prefer to name persistent state as a statement of fact rather than an imperative for a couple of reasons:

1. Statements of fact require less context.  If a future dev stumbles across a cookie called `hide_thank_donors` they may not be sure what causes this cookie to be set.  Saying `has_seen_thank_donors` is more cut-and-dry.
2. Statements of fact don't encode assumptions about how they will be used in the future.  Consider this hypothetical where we want to show a "find a course" dialog after you've seen thank donors:
   ```js
   // An imperative makes sense for one specific case but reads strangely in another
   if ( !cookies.get('hide_thank_donors') ) show('thank_donors')
   if ( cookies.get('hide_thank_donors') ) show('find_a_course')
   // A statement of fact makes sense for multiple uses
   if ( !cookies.get('has_seen_thank_donors') ) show('thank_donors')
   if ( cookies.get('has_seen_thank_donors') ) show('find_a_course')
   ````